pairinteraction_gui/app.py

```diff
@@ -3231,14 +3231,15 @@ def saveToZipfile(self, filename, idx):
                 data["numStates"] = [[v.shape[0]] for v in data["states"]]
 
             # save overlaps
-            if len(self.stateidx_field[idx]) == 1 and NO_BN in self.stateidx_field[idx]:
-                data["numOverlapvectors"] = self.stateidx_field[idx][NO_BN].shape[0]
-                data["overlapvectors"] = self.stateidx_field[idx][NO_BN]
-            elif len(self.stateidx_field[idx]) > 0:
-                _stateidx = self.stateidx_field[idx]
-                keys = [k for k in _stateidx.keys() if _stateidx[k] is not None]
-                data["numOverlapvectors"] = [[_stateidx[k].shape[0]] for k in keys]
-                data["overlapvectors"] = [[_stateidx[k]] for k in keys]
+            stateidx = self.stateidx_field[idx]
+            if len(stateidx) == 1 and NO_BN in stateidx:
+                if stateidx[NO_BN]:
```

Comment:
Sorry, this actually has to be
```suggestion
                if not stateidx[NO_BN] is None:
```
Otherwise Python tries to convert the numpy array to bool which numpy doesn't like.

Reply:
agreed, changed it again